hack/ci/Vagrantfile-ubuntu-local

```diff
@@ -0,0 +1,152 @@
+# -*- mode: ruby -*-
```

Comment:
Do we need this file?

Reply:
It was originally introduced for him to do local testing. I think it could be a nice addition... but let's then add that to a dedicated commit/PR.